addons/website/static/src/components/fields/fields.js

```diff
@@ -31,9 +32,20 @@ class PageUrlField extends UrlField {
         useEffect(
             (inputEl) => {
                 if (inputEl) {
-                    const fireChangeEvent = () => {
-                        inputEl.dispatchEvent(new Event("change"));
-                    };
+                    const originalValue = inputEl.value;
+                    let previousValueChanged = false;
+                    const fireChangeEvent = debounce(() => {
+                        const currentValue = inputEl.value;
+                        const valueChanged = currentValue !== originalValue;
+                        if (valueChanged !== previousValueChanged) {
+                            if (currentValue[0] !== '/') {
+                                inputEl.value = `/${currentValue}`;
+                            }
+                            inputEl.dispatchEvent(new Event("change"));
+                            inputEl.value = currentValue;
+                            previousValueChanged = valueChanged;
+                        }
+                    }, 300);
```

Comment:
I was testing one last time, and I think the delay could be smaller :thinking: 
Because before, the redirect apparition was quasi instant, but now we have a delay, and I think the goal was to have it directly 
I tried with 100 and it allows both to have the instant effect in normal connection and to avoid the jittering with slow connection
Maybe it could be even smaller I did not test, but it seems enough :slightly_smiling_face:
(if you change the value, don't forget to change it too on the test I think)

What do you think ? :slightly_smiling_face: